geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegionQueue.java

```diff
@@ -146,7 +146,7 @@ private void initializeEventSeqNumQueue() {
 
         if (logger.isDebugEnabled()) {
           logger.debug(
-              "For bucket {} ,total keys recovered are : {} last key recovered is : {} and the seqNo is ",
+              "For bucket: {} ,total keys recovered are: {} last key recovered is: {} and the seqNo is: {}",
```

Comment:
This: `{} ,` should be this: `{}, `. Also there are no commas in the rest of that message, so maybe the comma should be removed altogether.